raspimouse/src/raspimouse_component.cpp

```diff
@@ -484,6 +487,10 @@ void Raspimouse::stop_motors()
 
 void Raspimouse::calculate_odometry_from_pulse_counts(double & x, double & y, double & theta)
 {
+  double wheel_diameter = get_parameter(WHEEL_DIAMETER_PARAM).get_value<double>();
+  double wheel_tread = get_parameter(WHEEL_TREAD_PARAM).get_value<double>();
+  const auto PULSES_PER_REVOLUTION = get_parameter(PULSES_PER_REVOLUTION_PARAM).get_value<double>();
```

Comment:
定数を大文字化したいです。
関係するところも修正お願いします。

```suggestion
  const auto WHEEL_DIAMETER = get_parameter(WHEEL_DIAMETER_PARAM).get_value<double>();
  const auto WHEEL_TREAD = get_parameter(WHEEL_TREAD_PARAM).get_value<double>();
  const auto PULSES_PER_REVOLUTION = get_parameter(PULSES_PER_REVOLUTION_PARAM).get_value<double>();
```

Reply:
以下のコミットで対応しました。
5fd68b8